Assets/VRTK/Scripts/VRTK_InteractGrab.cs

```diff
@@ -37,6 +37,8 @@ public class VRTK_InteractGrab : MonoBehaviour
         public float grabPrecognition = 0f;
         [Tooltip("An amount to multiply the velocity of any objects being thrown. This can be useful when scaling up the `[CameraRig]` to simulate being able to throw items further.")]
         public float throwMultiplier = 1f;
+        [Tooltip("Automatically scale the throw velocity up and down to match the scale of the `[CameraRig]`.")]
+        public bool throwScalesWithPlayer = true;
```

Comment:
so do you think it's worth then just doing TransformVector and tell people if they want it to work the way it does now then they set their multiplier to like 0.7 or 0.5 to replicate that feeling?

I'd rather do that and see how it goes, I'm just loathed to keep adding options to things, look at the interactable object and pointer scripts, they're full of options and it makes my eyes hurt :(